doc/source/basics.rst

```diff
@@ -1755,12 +1755,14 @@ then the more *general* one will be used as the result of the operation.
    # conversion of dtypes
    df3.astype('float32').dtypes
 
+.. versionadded:: 0.20.0
```

Comment:
I'm a little confused. I tried using a dict with column/dtype mapping as an 
argument to astype and this works in 0.19.0:

```
>>> import numpy as np
>>> import pandas as pd
>>> pd.__version__
'0.19.0'
>>> dft = pd.DataFrame({'a': [1,0,1], 'b': [4,5,6], 'c': [7, 8, 9]})
>>> dft = dft.astype({'a': np.bool, 'c': np.uint8})
>>> dft
       a  b  c
0   True  4  7
1  False  5  8
2   True  6  9
>>> dft.dtypes
a     bool
b    int64
c    uint8
dtype: object
```

I think I'm missing what new functionality has been introduced in v0.20